Taskfile.yml

```diff
@@ -192,7 +192,6 @@ tasks:
       - test-integration-sqlite
       # no test-integration-hana
       - test-integration-mongodb
-
```

Comment:
CI fails because of that change. You can form this file locally with `task fmt-yaml`.

See also https://github.com/FerretDB/FerretDB/blob/c2b2c1d521a38daefb94a3be1cb473ded346c65f/CONTRIBUTING.md?plain=1#L359

Reply:
https://github.com/FerretDB/FerretDB/actions/runs/7218977191/job/19669213047?pr=3805